vj4/test/test_contest.py

```diff
@@ -148,37 +170,83 @@ def setUp(self):
   async def test_update_status_wrong_pid(self):
     rid = objectid.ObjectId()
     with self.assertRaises(error.ValidationError):
-      await contest.update_status(DOMAIN_ID_DUMMY, self.tid, ATTEND_UID, rid, 777, True, 100)
+      await contest.update_status(DOMAIN_ID_DUMMY, self.tid, ATTEND_UID, rid, 1000, True, 100)
```

Comment:
???

Reply:
777 is no longer a wrong pid (See https://github.com/vijos/vj4/pull/344/files#diff-15cf4000cd97361436e8ffa087659994R145)